EIPS/eip-4345.md

```diff
@@ -0,0 +1,59 @@
+---
+eip: 4345
+title: Difficulty Bomb Delay to April 2022
+author: Tim Beiko (@timbeiko)
+discussions-to: TBA
```

Comment:
Will need a discussion link before merging.

Reply:
That's fixed now :-)